### PR TITLE
gettextize: extract arguments to Qt's `tr`

### PR DESCRIPTION
**Pull request by ligfx**

### CODE REVIEW

**Comment by ligfx** [comment]
@JosJuice

**Comment by lioncash** [comment]
IMO © is better typographically here. `\u00A9` can be used instead if literals aren't desirable.

**Comment by ligfx** [comment]
@lioncash done!

**Comment by leoetlino** [comment]
Fwiw, xgettext has the `--qt` option:

    --qt   recognize Qt format strings (only language C++)

**Comment by leoetlino** [comment]
Never mind -- it doesn't do what you want. It just handles Qt's `arg` format strings and not `tr`.

File: Source/Core/DolphinQt2/AboutDialog.cpp [hunk 1]
```diff
@@ -54,8 +54,8 @@ AboutDialog::AboutDialog(QWidget* parent) : QDialog(parent)
   text_label->setOpenExternalLinks(true);
 
   QLabel* copyright =
-      new QLabel(tr("© 2003-%1 Dolphin Team. “GameCube” and “Wii” are"
-                    " trademarks of Nintendo. Dolphin is not affiliated with Nintendo in any way.")
+      new QLabel(tr("\u00A9 2003-%1 Dolphin Team. \u201cGameCubeu201d and \u201cWiiu201d are "
```
Comment (lioncash) [inline Source/Core/DolphinQt2/AboutDialog.cpp]: Late, but were the escaping backslashes accidentally left out after "GameCube" and "Wii" for `\u201D`?

Comment (leoetlino) [inline Source/Core/DolphinQt2/AboutDialog.cpp]: Oh whoops, I missed this, my bad! I'll open a PR to fix this

Comment (lioncash) [inline Source/Core/DolphinQt2/AboutDialog.cpp]: > my bad!

No worries, it happens =)

**Pull request merged**